test/functional/Valhalla/src_qtypes/org/openj9/test/lworld/ValueTypeTestClasses.java

```diff
@@ -46,7 +43,6 @@ public long getL() {
 		}
 	}
 
-	@ImplicitlyConstructible
 	static value class ValueTypePoint2D {
```

Comment:
https://bugs.openjdk.org/browse/JDK-8353746 says that:
> Tests using @NullRestricted annotated fields must be re-written to add the @Strict annotation and ensure the field is properly initialized with explicitly Java code according to the strict fields rules.
The @Strict annotation should be added to fields with @NullRestricted.